index.html

```diff
@@ -2450,6 +2450,46 @@ <h4>Enveloped Verifiable Credentials</h4>
 
         </section>
 
+        <section>
+          <h4>Enveloped Verifiable Presentations</h4>
+
+          <p>
+It is possible to express a [=verifiable presentation=] that has been secured
+using a securing mechanism that "envelopes" the payload, such as
```

Comment:
Distinctly, [`envelop` is probably what we want, rather than `envelope`](https://grammarist.com/spelling/envelop-envelope/). Assuming general concurrence, the last `e` gets removed from `envelope` and `envelopes`, but remains in `enveloped`; no change is needed for `enveloping`. Note that there *is* a pronunciation change ([accent on `ve` rather than `en`](https://www.vocabulary.com/articles/chooseyourwords/envelop-envelope/#:~:text=With%20enVElop%2C%20the%20accent%20is%20on%20the%20second%20syllable%2C%20while%20with%20ENvelope%2C%20the%20accent%20is%20on%20the%20first)).
```suggestion
using a securing mechanism that "envelops" the payload, such as
```